src/Vat.sol

```diff
@@ -312,6 +314,14 @@ contract Vat {
         emit Suck(u, v, rad);
     }
 
+    // --- Bridged DAI ---
+    function swell(address u, int256 rad) external auth {
+        dai[u] = _add(dai[u], rad);
+        surf   = surf + rad;
+
+        emit Swell(u, rad);
+    }
```

Comment:
`swell` should probably modify `debt` as well, although I need to look at the code more holistically to be sure.

Reply:
Corollary: `debt` would be `int` instead of `uint`.

Reply:
Yes that was our first reasoning. Keep `debt` as what actually is being backed by collateral from this network and have a separate field for what is being bridged. I think this way will keep things cleaner for the `end` otherwise you will need to count for it somehow at that stage.